components/src/core/info-list-item/info-list-item.component.ts

```diff
@@ -0,0 +1,58 @@
+import { Component, Input, ViewEncapsulation } from '@angular/core';
+
+@Component({
+    selector: 'pxb-info-list-item',
+    template: `
+        <mat-list-item
+            class="pxb-info-list-item"
+            [class.pxb-wrap]="wrapSubtitle || wrapTitle"
+            [class.pxb-dense]="dense"
+            [class.pxb-status]="statusColor"
+            [style.borderLeftColor]="statusColor"
+        >
+            <div mat-list-icon class="pxb-icon" [class.pxb-hide-padding]="hidePadding">
+                <ng-content select="[icon]"></ng-content>
+            </div>
+            <div class="pxb-left-component">
+                <ng-content select="[left-component]"></ng-content>
+            </div>
+            <div class="mat-body-1 pxb-title" matLine [class.pxb-wrap]="wrapTitle">{{ title }}</div>
```

Comment:
I think it would be best to leave any theme-related edits for the light/dark work.  I think the `mat-list-item` text is typically marked as a `matLine`.